include/bluetooth/mcc.h

```diff
@@ -500,12 +530,39 @@ int bt_mcc_read_segments_obj_id(struct bt_conn *conn);
 /** @brief Read Current Track Object ID */
 int bt_mcc_read_current_track_obj_id(struct bt_conn *conn);
 
+/** * @brief Set Current Track Object ID
+ *
+ * Set the Current Track to the the track given by the
+ * obj_id parameter
+ *
+ * @param obj_id    ID (in OTS) of the track to set as the current track
+ */
+int bt_mcc_set_current_track_obj_id(struct bt_conn *conn, uint64_t obj_id);
+
 /** @brief Read Next Track Object ID */
 int bt_mcc_read_next_track_obj_id(struct bt_conn *conn);
 
+/** * @brief Set Next Track Object ID
+ *
+ * Set the Next Track to the the track given by the
+ * obj_id parameter
+ *
+ * @param obj_id    ID (in OTS) of the track to set as the next track
+ */
+int bt_mcc_set_next_track_obj_id(struct bt_conn *conn, uint64_t obj_id);
```

Comment:
For consistency reasons, this renaming should be done for all functions (and callbacks) in the file.
It will of course also involve updating all users/tests of the module.
While useful, I am severely tempted to declare this out of scope for this PR.